packages/@n8n/nodes-langchain/nodes/chains/ChainLLM/methods/chainExecutor.ts

```diff
@@ -0,0 +1,102 @@
+import type { BaseLanguageModel } from '@langchain/core/language_models/base';
+import { StringOutputParser } from '@langchain/core/output_parsers';
+import type { ChatPromptTemplate, PromptTemplate } from '@langchain/core/prompts';
+import { CombiningOutputParser } from 'langchain/output_parsers';
+import type { IExecuteFunctions } from 'n8n-workflow';
+
+import type { N8nOutputParser } from '@utils/output_parsers/N8nOutputParser';
+import { getTracingConfig } from '@utils/tracing';
+
+import { createPromptTemplate } from './promptUtils';
+import type { ChainExecutionParams } from './types';
+
+/**
+ * Determines which output parser to use based on the number of parsers provided
+ */
+function getOutputParser(outputParsers: N8nOutputParser[]): N8nOutputParser {
+	if (outputParsers.length === 1) {
+		return outputParsers[0];
+	}
+
+	// If multiple parsers, combine them
+	return new CombiningOutputParser(...outputParsers) as unknown as N8nOutputParser;
```

Comment:
I found this use case a bit unreliable (something is off, at least on a logs panel). I understand it was already there before. Just wonder do we need to support this at all? Can we remove it and limit with a single parser? Seems like super-rare use-case.

Reply:
Good call. I limited all the output parsers to a single connection, also in agents.